src/handler.js

```diff
@@ -15,6 +15,35 @@ function homeHandler(req, res){
   })
 }
 
+//Create handler for main.js and main.css
+function assetsHandler(req, res){
+  //Handler Variables
+  var url = req.url;
+  console.log(url);
+  var extension = url.split('.')[1];
+  var extensionType = {
```

Comment:
Can move this out of the function, just have it at the top of the page so we don't make this variable everytime the assetHandler function is called.